code/modules/power/tesla/coil.dm

```diff
@@ -95,5 +115,15 @@
 
 	return ..()
 
+/obj/machinery/power/grounding_rod/attack_hand(mob/user)
+	if(can_buckle && isliving(user.pulling) && user.a_intent == INTENT_GRAB && !has_buckled_mobs())
```

Comment:
@RemieRichards 
Fun fact: `forceMove` UNBUCKLES PEOPLE!
Using it after `user_buckle_mob`? What a wonderful fucking idea! Spent half a hour debugging this shit!